src/components/index.js

```diff
@@ -0,0 +1,6 @@
+
+import TrustedCompanies from './TrustedCompanies/TrustedCompanies';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
import TrustedCompanies from "./TrustedCompanies/TrustedCompanies";
```